### ISSUE TITLE
💅 --log-kind=compact is still printing pretty

### ISSUE THREAD

**Issue by suthakarvks** [biomejs/biome#8181]
### Environment information

```bash
CLI:
  Version:                      2.3.6
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           unset
  JS_RUNTIME_NAME:              unset
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false

Linter:
  JavaScript enabled:           unset
  JSON enabled:                 unset
  CSS enabled:                  unset
  GraphQL enabled:              unset
  Recommended:                  true
```

### Rule name

--log-kind

### Playground link

none

### Expected result

I want to see the compact warnings instead pretty print. Using --log-kind=compact or json both are not working. --diagnosis-level=error is working as expected. 

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by siketyan** [comment]
Assuming you want compact _diagnostics_, which shows the lint errors. Logs and diagnostics are different; logs are used for internal and they aren't usually printed to the terminal unless you use the --log-level option.

**Comment by suthakarvks** [comment]
> Hello [@suthakarvks](https://github.com/suthakarvks), please provide a minimal reproduction. You can use one of the following options:
> 
> * Provide a link to [our playground](https://biomejs.dev/playground), if it's applicable.
> * Provide a link to GitHub repository. To easily create a reproduction, you can use our interactive CLI via `npm create @biomejs/biome-reproduction`
> 
> Issues marked with `S-Needs repro` will be **closed** if they have **no activity within 3 days**.

Here is the [ playground link ](https://biomejs.dev/playground/?quoteStyle=single&jsxQuoteStyle=single&lintRules=all&code=ZgB1AG4AYwB0AGkAbwBuACAAbwBuAGUAKAB0AG8AbwBsACkAIAB7AAoAIAAgAGMAbwBuAHMAbwBsAGUALgBsAG8AZwAoACIAVgBhAHIAaQBhAGIAbABlACAAaQBzACAAbgBvAHQAIAB1AHMAZQBkACIAKQA7AAoAIAAgAHIAZQB0AHUAcgBuACAAZgBhAGwAcwBlADsACgB9AAoA&language=js)

If you open console, you will see as below,

`/main.js:2:3 [lint/suspicious/noConsole](https://biomejs.dev/linter/rules/no-console)  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Don't use console.
  
    1 │ function one(tool) {
  > 2 │   console.log("Variable is not used");
      │   ^^^^^^^^^^^
    3 │   return false;
    4 │ }
  
  ℹ The use of console is often reserved for debugging.
  
  ℹ Unsafe fix: Remove console.
  
    1 1 │   function one(tool) {
    2   │ - ··console.log("Variable·is·not·used");
    3 2 │     return false;
    4 3 │   }`

I want to print this in compact way so that it is not taking lot of space and hide important watnings/errors

**Comment by suthakarvks** [comment]
> Assuming you want compact _diagnostics_, which shows the lint errors. Logs and diagnostics are different; logs are used for internal and they aren't usually printed to the terminal unless you use the --log-level option.

Thanks for the clarification, I just need a compact warnings instead of giving too much details on each warnings. I have giving the reference above.

**Comment by ematipico** [comment]
There's a task for that which isn't implemented yet.

Subscribe to the issue https://github.com/biomejs/biome/issues/6970

I'll close this issue
